internal/wrr/random.go

```diff
@@ -19,15 +19,21 @@ package wrr
 
 import (
 	"fmt"
+	"sort"
 	"sync"
 
 	"google.golang.org/grpc/internal/grpcrand"
 )
 
 // weightedItem is a wrapped weighted item that is used to implement weighted random algorithm.
 type weightedItem struct {
-	Item   interface{}
-	Weight int64
+	Item interface{}
+	// TODO Delete Weight? This field is not necessary for randomWRR to work.
```

Comment:
Delete this field if it's not used.

And it seems all fields in this struct can be unexported, right? Can you also change that? Thanks

Reply:
done